.github/workflows/continuous-integration-workflow.yml

```diff
@@ -49,11 +49,16 @@ jobs:
             cxx: 'clang++'
             cmake_build_type: 'RelWithDebInfo'
             backend: 'THREADS'
-            clang-tidy: '-DCMAKE_CXX_CLANG_TIDY="clang-tidy;-warnings-as-errors=*"'
+            cmake_extra_args: '-DCMAKE_CXX_CLANG_TIDY="clang-tidy;-warnings-as-errors=*"'
```

Comment:
why this change?

Reply:
To avoid having separate variables for `clang-tidy` and '-DBUILD_SHARED_LIBS=ON'.